test/plugins/individual-collateral/compoundv3/CusdcV3Wrapper.test.ts

```diff
@@ -598,6 +598,18 @@ describeFork('Wrapped CUSDCv3', () => {
       )
     })
 
+    it('regression test: able to claim rewards even when they are big without overflow', async () => {
+      // Nov 28 2023: uint64 math in CusdcV3Wrapper contract results in overflow when COMP rewards are even moderately large
+
+      const compToken = <ERC20Mock>await ethers.getContractAt('ERC20Mock', COMP)
+      expect(await compToken.balanceOf(wcusdcV3.address)).to.equal(0)
+      await advanceTime(1000)
+      await enableRewardsAccrual(cusdcV3, bn('2e18')) // enough to revert on uint64 implementation
```

Comment:
Can confirm this fails, so seems good. I also like the `2e18` example because it's under the `uint64` limit and hence highlights where the issue comes from.